server/graphql/v2/mutation/MemberInvitationMutations.js

```diff
@@ -48,19 +50,24 @@ const memberInvitationMutations = {
 
       memberAccount = await fetchAccountWithReference(memberAccount, { throwIfMissing: true });
       account = await fetchAccountWithReference(account, { throwIfMissing: true });
-
-      if (!req.remoteUser.isAdminOfCollective(account)) {
+      const memberInfo = pick(args, ['role', 'description', 'since']);
+
+      if (args.role === roles.CONNECTED_COLLECTIVE && req.remoteUser.isRoot()) {
+        // Special case: when users are root, we allow them to create connected collectives directly
+        await models.Member.connectCollectives(memberAccount, account, req.remoteUser, memberInfo);
+        purgeCacheForCollective(account.slug);
+        purgeCacheForCollective(memberAccount.slug);
+        return null;
```

Comment:
A bit weird to return `null` here? Should we use a different mutation instead?

Reply:
As per https://github.com/opencollective/opencollective-frontend/pull/6825#issuecomment-913460330, I agree that using this mutation seems a bit hacky. Should I go with a (root-only) new `createMember` mutation?